CAM Terrafrom supports 2 additional providers (#2801)
CenturyLinkCloud/PublicKB: Cloud Application Manager/Automating Deployments/template-box.md

> }}}

Reviewer: Please, update metadata values (date, author)

**In this article:**

> - [Overview](#overview)

Reviewer: Please, keep the asterisks for the ToC bullet points

### Deploy Using Terraform Templates

A Terraform Template box allows you to apply a **Terraform version 0.11** configuration consisting of one or many files on the following Providers:
-  Amazon Web Services
-  CenturyLink Cloud
-  CenturyLink Private Cloud on VMware Cloud Foundation
-  Google Cloud Platform
-  Microsoft Azure
-  VMware vCenter/vSphere
-  VMware vCloud Director

This feature provides you with the ability to use the power of Cloud Application Manager (instance history, Lifecycle Editor, bindings, box versioning) combined with any service supported by Terraform.

#### To learn more about Terraform

> In order to understand Terraform's configurations language see [Terraform 0.11 documentation here](https://www.terraform.io/docs/configuration-0-11/index.html). Currently Cloud Application Manager can deploy **Terraform version 0.11** templates.

Reviewer: ```suggestion
In order to understand Terraform's configurations language see [Terraform 0.11 documentation here](https://www.terraform.io/docs/configuration-0-11/index.html). Currently, Cloud Application Manager supports the deployment of **Terraform version 0.11** templates.
```
